From 6a051666b50ae1df24b24312a7a56a03e343c15d Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Fri, 16 Aug 2019 09:48:21 +0300
Subject: [PATCH] sample: fix readline include in ipmi_serial_bmc_emu.c

Bundled editline has been removed. Use the readline headers instead. Fix
the build when editline is not installed on host.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Upstream status: sent to openipmi-developer@lists.sourceforge.net (moderated)

 sample/ipmi_serial_bmc_emu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sample/ipmi_serial_bmc_emu.c b/sample/ipmi_serial_bmc_emu.c
index e0ae0197688b..1b8bb9ac201f 100644
--- a/sample/ipmi_serial_bmc_emu.c
+++ b/sample/ipmi_serial_bmc_emu.c
@@ -42,7 +42,7 @@
 #include <unistd.h>
 #include <stdlib.h>
 #include <sys/select.h>
-#include <editline/readline.h>
+#include <readline/readline.h>
 
 #define _GNU_SOURCE
 #include <getopt.h>
-- 
2.23.0.rc1

